app/Http/Controllers/admin/SettingsController.php

```diff
@@ -144,9 +143,6 @@ public function updateGeneral(Request $request)
 
       $inputImages = Input::only('header_logo', 'footer_logo', 'nominate_image', 'background_image');
 
-      // $this->settingsForm->validate($inputText);
```

Comment:
@angaither took me a while to figure out why this was showing up here but not on my local branch...finally realized it ~is~ removed